src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./Navbar.scss";
+
+function Navbar() {
+  return (
+    <section className="header-container">
+      <div className="header-wrapper">
+        <div className="navbar-container">
+          <div className="menu-container">
+            <img src="src/assets/images/HamburgIcon.svg" alt="hambur" />
+          </div>
+          <div className="logo-container">
+            <img src="src/assets/images/logo.svg" alt="logo" />
+          </div>
+          <div className="navbar-buttons-container">
+            <div className="epicure-text-title">EPICURE</div>
```

Comment:
Maybe save all of the important constants, such as the title of your app `epicure`, of this app in a globally available reusable object.